src/Nethermind/Nethermind.Synchronization/StateSync/PeerInfoExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Synchronization.Peers;
+
+namespace Nethermind.Synchronization.StateSync;
+
+public static class PeerInfoExtensions
+{
+    public static bool CanGetNodeData(this PeerInfo peerInfo)
+    {
+        return peerInfo.SyncPeer.Node.EthDetails == "eth66";
```

Comment:
Maybe let's change that from string to some object that has version explicitly?

Reply:
Done